### DISCUSSION TITLE
Description of why developed

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#554] [Ideas]
Maybe there can be a better explanation, for example, 
1. "We aim to regain the dignity of being a computer user, and insist on using a compact design and a GUI that users can universally understand."
2. "In order to reduce the threshold for users to understand icons, we use realistic icons to make the system look closer to users and more life-like."
3. "We are committed to designing a BSD system that is very easy for users to get started in all BSD systems. Users do not need coding or instructions, only a mouse."

Maybe you can refer to it.

**Comment by probonopd** [comment]
As far as I am concerned, I have stated my reasons [here](https://medium.com/@probonopd/the-desktop-metaphor-must-be-restored-its-under-attack-6fcd8fa45cc).

**Comment by louies0623** [comment]
Actually, what I mean is the explanatory text below, in the Way part.
https://github.com/helloSystem/hello
